server/controllers/services/email.js

```diff
@@ -207,7 +220,30 @@ module.exports = (app) => {
         return Promise.map(users, (user) => emailLib.send('email.approve', `members@${slug}.opencollective.com`, getData(user), {bcc:user.email}));
       })
       .then(() => res.send('Mailgun webhook processed successfully'))
-      .catch(next);
+      .catch(e => {
+        switch (e.message) {
+          case 'no_subscribers':
```

Comment:
but it's only in one file. Honestly, in the age of modern IDE and autocomplete, it doesn't really matter if it's in one single file.